folly/Subprocess.cpp

```diff
@@ -560,8 +560,13 @@ int Subprocess::prepareChild(
   // If requested, close all other file descriptors.  Don't close
   // any fds in options.fdActions_, and don't touch stdin, stdout, stderr.
   // Ignore errors.
+  #ifdef __ANDROID__
+  int tableSize = sysconf(_SC_OPEN_MAX);
+  #else
+  int tableSize = getdtablesize();
```

Comment:
Any particular reason to keep the call to `getdtablesize` on non-Android platforms, seeing as it's less portable?

Reply:
I've never really tested it on **all** other platforms - I just checked android and it works on it. To change it for every platform I should check every one of them, don't you think?